src/Codeception/Module/WebDriver.php

```diff
@@ -2177,20 +2179,52 @@ public function dragAndDrop($source, $target)
      *
      * @throws \Codeception\Exception\ElementNotFound
      */
-    public function moveMouseOver($cssOrXPath, $offsetX = null, $offsetY = null)
+    public function moveMouseOver($cssOrXPath = null, $offsetX = null, $offsetY = null)
     {
-        $el = $this->matchFirstOrFail($this->webDriver, $cssOrXPath);
+        if (null === $cssOrXPath) {
+            $el = $this->matchFirstOrFail($this->webDriver, $cssOrXPath);
+        }
+
         $this->webDriver->getMouse()->mouseMove($el->getCoordinates(), $offsetX, $offsetY);
     }
 
     /**
-     * Performs contextual click with the right mouse button on an element.
+     * Performs contextual click with the right mouse button on an element or page.
```

Comment:
Ok, please update the description of this method. As you see it says "right mouse button"